module1/lessons/Week1/WrapUp.md

```diff
@@ -0,0 +1,20 @@
+# Daily Wrap-Up Meeting
+
+## What x Why
+
+The Daily Wrap-Up Meeting is an opportunity to meet with your Instructors and the entire cohort at the end of each day. This meeting will be held **Monday through Wednesday, 3:45pm MT**.
+
+## Summary
+
+In general, the following are covered in the Daily Wrap-Up:
+
+* Overview of deliverables due that day
+* The relationship of the day's lesson or lab to a larger deliverable (i.e. the mod Project)
+* Q&A for lessons and labs covered that day or related to the day
```

Comment:
We might want a note here that if a question is lengthy, we will take note and build in time to answer (or create a slack post to answer the question)

Reply:
Thanks. I added a bullet.